languages/python/default.nix

```diff
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
+      type = with lib.types; nullOr lines;
+      description = ''
+        Python interpreter startup configuration. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        import numpy as np
+      '';
+    };
+    configPath = lib.mkOption {
+      type = lib.types.path;
+      description = ''
+        Python interpreter startup configuration file path. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = "${config.xdg.configHome}/python/startup.py";
+      defaultText = "$XDG_CONFIG_HOME/python/startup.py";
+      example =
+        lib.literalExpression
+        ''"${config.xdg.configHome}/pythonrc.py"'';
+    };
+    historyPath = lib.mkOption {
+      type = with lib.types; nullOr path;
+      description = ''
+        Python interpreter history file path. See
+        <https://docs.python.org/3.13/using/cmdline.html#envvar-PYTHON_HISTORY>
```

Comment:
I’m not that familiar with the markdown conventions in NixOS module code but I’m also not sure if `<…>` is required/well-placed here.

Reply:
Not really familiar either but [it seems to be the convention for Home Manager](https://github.com/nix-community/home-manager/blob/b2f56952074cb46e93902ecaabfb04dd93733434/modules/programs/alacritty.nix#L45).

Reply:
Ah OK, thanks!